src/eslint/index.js

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Here and below it complains about `index.js` file coverage which we do not expect to load in tests anyway.

How about we skip the check for this PR